README.md

```diff
@@ -47,6 +47,10 @@ In this workflow switch owner verifies device's Initial Attestation Key (IAK) an
 fully control certificate structure, revocation and expiration policies and (2) remove external dependency on switch vendor CA during TPM attestation workflow. The assumption is that before the device is shipped to the switch owner, a switch vendor provisions each control card with IAK and IDevID certificates following the TCG specification in
 [Section 5.2](https://trustedcomputinggroup.org/wp-content/uploads/TPM-2p0-Keys-for-Device-Identity-and-Attestation_v1_r12_pub10082021.pdf#page=20) and [Section 6.2](https://trustedcomputinggroup.org/wp-content/uploads/TPM-2p0-Keys-for-Device-Identity-and-Attestation_v1_r12_pub10082021.pdf#page=30).
 
+Switch vendors should use the strongest asymmetric cryptographic algorithms supported by the TPM 2.0 to create IAK and IDevID key pairs. The algorithms must be at least ECC P256 or, less preferably, RSA 2048.
+The most preferable and recommended approach, however, would be to rely on ECC P521 or, less preferably, RSA 4096.
```

Comment:
is there a reason we wouldn't just say: "accept 521 / rsa4096 only"

Given a chance to buy a hardware/chip for our purposes, we'd want the answer to be the very best available 'right now', since that chip is going to live with us for ~10 yrs.

Reply:
I used this wording, because this requirement came a bit late for vendors and additionally may sound a bit too strict in the context of the broader openconfig initiative. I am happy to change it to "must be ECC P521 or, less preferably, RSA 4096" though!

Reply:
I updated the requirements, so let me know what you think!